### PR TITLE
Update to jenkins 2.5.1

### PR DESCRIPTION
**Pull request by rtnpro**
Updating Jenkins version to 2.5.1 to work around slow installation on Jenkins.

### CODE REVIEW

**Comment by rtnpro** [comment]
#dotests-debug

File: provisions/roles/jenkins/master/tasks/ci.yml [hunk 1]
```diff
@@ -17,11 +17,17 @@
   when: test
   sudo: yes
 
+- name: wait for 10 seconds
+  pause:
+    seconds: 10
+
```
Comment (navidshaikh) [inline provisions/roles/jenkins/master/tasks/ci.yml]: This wait is to let Jenkins create test job or something else? If yes, we are already doing 50 retries with 10 seconds delay. Please add comment with reason we are waiting for 10 seconds.

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

File: provisions/roles/jenkins/master/tasks/ci.yml (hunk 1 above)
Comment (rtnpro) [inline provisions/roles/jenkins/master/tasks/ci.yml]: @navidshaikh I was testing out something with delays. It has now been removed.

**Comment by bamachrn** [comment]
LGTM, this will minimize the deployment time

**Pull request merged**